boards/lilygo/twatch_s3/doc/index.rst

```diff
@@ -0,0 +1,189 @@
+.. zephyr:board:: twatch_s3
+
+Overview
+********
+
+LILYGO T-Watch S3 is an ESP32-S3 based smartwatch with the following features:
+
+- ESP32-S3-R8 chip
+   - Dual core Xtensa LX-7 up to 240MHz
+   - 8 MB of integrated PSRAM
+   - Bluetooth LE v5.0
+   - Wi-Fi 802.11 b/g/n
+- 16 MB external QSPI flash (Winbond W25Q128JWPIQ)
+- Power Management Unit (X-Powers AXP2101) which provides
+   - Regulators (DC-DCs and LDOs)
+   - Battery charging
+- 470 mAh battery
+- RTC (NXP PCF8563)
+- Haptic (Texas Instruments DRV2605)
+- Accelerometer (Bosh BMA423)
+- 240x240 pixels LCD with touchscreen
+   - ST7789V LCD Controller
+   - Focaltech FT5336 touch sensor
+- Microphone (Knowles SPM1423HM4H-B)
+- LoRA radio (Semtech SX1262)
```

Comment:
Would be cool to add the corresponding dts node :)

Reply:
I saw that there is the `ttgo_lora32` board which features the same LoRA controller, so it very likely that I can copy-paste the DTS configuration from there. However before doing that I would like to test this to be sure that what I'm adding is reasonably correct.